### DISCUSSION TITLE
random_colors flag in dipy_horizon does not work as before

### DISCUSSION THREAD

**Discussion by drombas** [dipy/dipy#2465] [Scientific Visualization] [answered]
## Description
When trying to use  `--random_colors` flag in `dipy_horizon` to visualize two bundles I get the following error:

![image](https://user-images.githubusercontent.com/50577357/128494038-972e3d0f-4748-4446-b9ae-1f48058eaffe.png)

I saw that the usage of this argument was extended in #2374. In fact, if I use `--random_colors tracts` it works well so not sure if this is a bug or something expected.

**Comment by skoudoro** [comment]
Thank you for your question. @guaje will answer you soon!

**Comment by guaje** [comment, marked as answer]
Hi @drombas, thanks for catching this.

I ran the command `dipy_horizon --random_colors AF_L.trk CST_L.trk` and it works perfectly. The problem occurs when you put the argument at the end and don't select one of the available options. Use this form in the meantime while I fix this.

**Comment by drombas** [comment]
Thanks, @guaje!

**Comment by skoudoro** [comment]
There is a solution to this question. I will convert this issue to a discussion and close it
